frontend/src/routes/Video.tsx

```diff
@@ -882,7 +882,7 @@ const Metadata: React.FC<MetadataProps> = ({ event, realmPath }) => {
                 <VideoDate {...{ event }} />
             </div>
             {/* Buttons */}
-            <div css={{
+            <section aria-label={t("video.manage-buttons")} css={{
```

Comment:
Making a `section` out of that makes sense to me. Not sure if I would call it "Video management" though. We have an optional "Download" button, a share button and an conditional "Manage" button in there. Maybe just "video actions" or "video action buttons" or sth like that?